src/style.css

```diff
@@ -1 +1,74 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  color: #000;
+}
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 50px;
+}
+
+.logo {
+  display: flex;
+  margin: 10px 0;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  height: 100%;
+}
+
+.nav__item {
+  height: 100%;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  height: 100%;
```

Comment:
use paddings

Reply:
?

Reply:
In this situation better use height and flex links, cause paddings in mockup not int number (as like 23.5) + perfect pixel test